target/linux/ramips/dts/mt7628an_zyxel_keenetic-4g.dts

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-4g", "mediatek,mt7628an-soc";
+	model = "Keenetic 4G";
+
+	aliases {
+		led-boot = "/leds/power";
+		led-failsafe = "/leds/power";
+		led-running = "/leds/power";
+		led-upgrade = "/leds/power";
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		
+		internet {
+			label = "keenetic-4g:green:internet";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+
+		
+		wifi {
+			label = "keenetic-4g:green:wifi";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+		};
+		
+    };
+	
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+            label = "reset";
+            gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+        };
+
+
+		wps {
```

Comment:
Please take care of formatting/indent:
- DTS should have only tabs for indent. One empty line should be left between nodes.

Reply:
Fix tabs